dep11/component.py

```diff
@@ -30,10 +30,11 @@
 
 ###########################################################################
 DEP11_VERSION = "0.8"
-time_str = str(datetime.date.today())
+now = datetime.datetime.now(datetime.timezone.utc).isoformat()
```

Comment:
FYI it's this and not utcnow() because this way you get the timezone set in the output.